Source/Extensions/Blazorise.TreeView/Internal/_TreeViewNode.razor

```diff
@@ -27,9 +27,9 @@
                 }
 
                 <_TreeViewNodeContent TNode="TNode" NodeState="@nodeState"
-                                      NodeStyling="@(ResolveNodeStylingAction(NodeStyling))"
+                                      NodeStyling="@NodeStyling"
                                       SelectionMode="@SelectionMode"
-                                      SelectedNodeStyling="@(ResolveSelectedNodeStylingAction(SelectedNodeStyling))">
+                                      SelectedNodeStyling="@SelectedNodeStyling">
```

Comment:
The commit only has this replacement, is there nothing else we need to do? Because previously it would have an Action with an empty return? If everything ok in case these are null?

Reply:
We don't need it as far as I can tell. I really don't know why the person who created the first TreeView version did that but as long as we don't invoke on an event that is null we are good.